src/main/java/org/mockito/internal/invocation/finder/AllInvocationsFinder.java

```diff
@@ -32,9 +34,33 @@ private AllInvocationsFinder() {}
         return new LinkedList<Invocation>(invocationsInOrder);
     }
 
+    /**
+     * Gets all stubbings from mocks. Invocations are ordered earlier first.
+     *
+     * @param mocks mocks
+     * @return stubbings
+     */
+    public static Set<StubbedInvocationMatcher> findStubbings(Iterable<?> mocks) {
```

Comment:
Idea or later improvement, `StubbedInvocationMatcher` is a class, what do you think of introducing a sub-interface of `InvocationOnMock` that could called `StubbedInvocation` in order to hide the